app/assets/src/components/ui/controls/PlusMinusControl.jsx

```diff
@@ -0,0 +1,35 @@
+import cx from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import PlusControlIcon from "~/components/ui/icons/PlusControlIcon.jsx";
+import MinusControlIcon from "~/components/ui/icons/MinusControlIcon.jsx";
+import cs from "./plus_minus_control.scss";
+
+const PlusMinusControl = ({
+  onClickPlus,
```

Comment:
nit: we tend to follow noun-verb i.e. `onPlusClick` (https://jaketrent.com/post/naming-event-handlers-react/)